arch/arm/src/rtl8720c/amebaz_driver.c

```diff
@@ -112,7 +112,7 @@ static void amebaz_state_deinit(struct amebaz_state_s *state)
 
 static int amebaz_state_init(struct amebaz_state_s *state)
 {
-  if (nxsem_init(&state->mutex, 0, 0) != OK)
+  if (nxsem_init(&state->sem, 0, 0) != OK)
     {
       return -ENOMEM;
     }
```

Comment:
```suggestion
  if (nxsem_init(&state->sem, 0, 0) != OK)
    {
      return -ENOMEM;
    }
    
    nxsem_set_protocol(&state->sem, SEM_PRIO_NONE);
```
Also I do not think that `nxsem_init` can ever fail here. The error check can be removed

Reply:
Done.